__tests__/helpers/mockServerController.ts

```diff
@@ -1,53 +1,33 @@
-const { spawn } = require("child_process");
+import mockserver, { IServerHandle } from "../../src/server";
```

Comment:
with typescript we can setup local module resolution and stop doing relative paths. What do you think?

Reply:
I tried it, but it is not possible.

Reply:
Tell me more :)

Reply:
You can set different roots with typescript, but that just works with modules, so I would need to write `from "server"` instead, which is way too much magic IMHO. Here is the handbook article I read for that: http://www.typescriptlang.org/docs/handbook/module-resolution.html

Reply:
@nLight is this enough info for you or would you like to get it working together today?